### DISCUSSION TITLE
New vulnerabilities notification

### DISCUSSION THREAD

**Discussion by st1ckyf1ng4z** [DependencyTrack/dependency-track#2709] [Q&A] [answered]
Hello Community!
Tell me please, if it's possible to send single email notification about new vulnerabilities in the project?
Now it's a separate message for each vulnerability.
Can this be done by custom template?

**Comment by syalioune** [comment, marked as answer]
Hello @st1ckyf1ng4z 
Notification aggregation is currently not possible.

**Reply by st1ckyf1ng4z** [reply]
Thanks for the answer

**Comment by msymons** [comment]
@st1ckyf1ng4z,  functionality to support notification batching is something that I requested in #322.

Note this comment...

> As a workaround, it would likely be elementary to consume webhooks external to the system, store then for 24 hours, and send out summaries by reading the the payloads received since the last purge.
